lib/jxl/modular/transform/palette.h

```diff
@@ -445,6 +446,44 @@ static Status FwdPalette(Image &input, uint32_t begin_c, uint32_t end_c,
   std::vector<pixel_type> color(nb);
   std::vector<float> color_with_error(nb);
   std::vector<const pixel_type *> p_in(nb);
+
+  if(lossy) {
+    // Map colors to their frequencies in the image.
```

Comment:
This comment doesn't really seem to match what is going on here (I believe we're only counting colors that have an equal 1-cross around them?)

Reply:
Changed the comment.